packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -65,14 +54,15 @@ class PointerEventConverter {
     );
   }
 
-  /// Expand the given packet of pointer data into a sequence of framework
+  /// format the given packet of pointer data into a framework
   /// pointer events.
   ///
   /// The `devicePixelRatio` argument (usually given the value from
   /// [dart:ui.Window.devicePixelRatio]) is used to convert the incoming data
   /// from physical coordinates to logical pixels. See the discussion at
   /// [PointerEvent] for more details on the [PointerEvent] coordinate space.
-  static Iterable<PointerEvent> expand(Iterable<ui.PointerData> data, double devicePixelRatio) sync* {
+  static List<PointerEvent> format(List<ui.PointerData> data, double devicePixelRatio) {
```

Comment:
Is there a reason why the generator needs to be replaced by a list?

Reply:
if the result is 1 to 1 mapping, we prefer to use list

Reply:
why?

Reply:
I took that from @goderbauer 's suggestion. Should I change it back?

Reply:
fixed